overlord/hookstate/hookmgr.go

```diff
@@ -310,7 +345,7 @@ func runHookAndWait(snapName string, revision snap.Revision, hookName, hookConte
 
 	// Hook was aborted.
 	case <-tomb.Dying():
-		if err := command.Process.Kill(); err != nil {
+		if err := killemAll(command); err != nil {
```

Comment:
we should call killTimer.Stop() here to avoid double killing and failing one of the times, also maybe ignore killemAll errors if the killTimer.Stop return bool tells us the timer was already triggered